src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+  Hello, world!
+  </body>
+</html>
```

Comment:
check the checklist, there should be an empty line at the end of the file
(the checklist is in the root folder of the project)